examples/simple_example.rs

```diff
@@ -0,0 +1,136 @@
+//! Centralized Telescope example with BLS signatures
+
+use alba::centralized_telescope::params::Params;
+use alba::centralized_telescope::proof::Proof;
+use alba::centralized_telescope::CentralizedTelescope;
+use blst::min_sig::{AggregatePublicKey, AggregateSignature, PublicKey, SecretKey, Signature};
+use blst::BLST_ERROR;
+use rand_chacha::ChaCha20Rng;
+use rand_core::{CryptoRng, RngCore, SeedableRng};
+use std::collections::HashMap;
+
+const DATA_LENGTH: usize = 48;
+pub(crate) type Element = [u8; DATA_LENGTH];
+
+struct AlbaSigner {
+    signing_key: SecretKey,
+    verification_key: PublicKey,
+}
+
+struct ThresholdSignature {
+    proof: Proof,
+    key_list: Vec<PublicKey>,
+}
+
+impl AlbaSigner {
+    fn new(rng: &mut (impl RngCore + CryptoRng)) -> Self {
+        let mut ikm = [0u8; 32];
```

Comment:
What does `ikm` stands for? Can you rename it buffer if it is not standard instead?

Reply:
I was more curious than anything :)